rackspace-server-manager.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+'''
+A script to create and delete the number of servers on Rackspace
+'''
+
+import pyrax
+import time
+import re
+import os
+import argparse
+from Crypto.PublicKey import RSA
+
+def create_key():
+    key = RSA.generate(1024)
+    f = open("private.pem", "wb")
+    f.write(key.exportKey('PEM'))
+    f.close()
+
+    pubkey = key.publickey()
+    f = open("public.pem", "wb")
+    f.write(pubkey.exportKey('OpenSSH'))
+    f.close()
+
+
+def create_node(nova, counts):
+    flavor = nova.flavors.find(name='2 GB General Purpose v1')
+    image = nova.images.find(name='centos7-test')
+    create_key()
+    pubkey = open('public.pem', 'r').read()
+    nova.keypairs.create('distkey', pubkey)
+    for count in range(int(counts)):
+        name = 'distributed-testing.'+str(count+1)
+        node = nova.servers.create(name=name, flavor=flavor.id,
+                                   image=image.id, key_name='distkey')
+
+        while node.status == 'BUILD':
+            time.sleep(5)
+            node = nova.servers.get(node.id)
+
+        ip_address = None
+        for network in node.networks['public']:
+            if re.match('\d+\.\d+\.\d+\.\d+', network):
```

Comment:
Use this data to create an ansible hosts file in the right format. Something like this:


    81c5df65-0c80-48ee-9eaf-a8b5bb947aef.cloud.gluster.org ansible_hostname=192.168.1.1
    c6bd9cc5-254f-4240-bf5a-7290ab2d3433.cloud.gluster.org ansible_hostname=192.168.1.2